roles/provision_vcenter/tasks/deploy_from_iso.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Template Installer Config JSON
+  ansible.builtin.template:
+    src: vcsa_deployment_config.json.j2
+    dest: /tmp/vcsa_deployment_config.json
```

Comment:
If I understand correctly this file contains password, so I would make sure it's always removed.

Reply:
Also I think  `mode: '0744'` is not OK here, since it contains passwords. Can it be 600?